test/adapter/json/collection_test.rb

```diff
@@ -75,6 +75,16 @@ def test_root_is_underscored
 
             assert_equal 1, adapter.serializable_hash[:virtual_values].length
           end
+
+          def test_include_option
+            serializer = ArraySerializer.new([@first_post, @second_post])
+            adapter = ActiveModel::Serializer::Adapter::Json.new(serializer, include: '')
```

Comment:
It did so every time for the Json adapter, and only for collections for the Attribute adapter.